tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
+# requesting ``'left'`` will select all epochs with Event IDs 1 and 3). An
+# example of this is shown in the next section. There is also a convenient
+# :func:`~mne.viz.plot_events` function for visualizing the distribution of
+# events across the duration of the recording (to make sure event detection
+# worked as expected). Here we'll also make use of the :class:`~mne.Info`
+# attribute to get the sampling frequency of the recording (so our x-axis will
+# be in seconds instead of in samples).
+
+fig = mne.viz.plot_events(events, event_id=event_dict, sfreq=raw.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
+###############################################################################
+# Epoching continuous data
+# ^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Define parameters:
-
-import numpy as np  # noqa
-n_cycles = 2  # number of cycles in Morlet wavelet
-freqs = np.arange(7, 30, 3)  # frequencies of interest
-
-##############################################################################
-# Compute induced power and phase-locking values and plot gradiometers:
-
-from mne.time_frequency import tfr_morlet  # noqa
-power, itc = tfr_morlet(epochs, freqs=freqs, n_cycles=n_cycles,
-                        return_itc=True, decim=3, n_jobs=1)
-power.plot([power.ch_names.index('MEG 1332')])
-
-##############################################################################
-# Inverse modeling: MNE and dSPM on evoked and raw data
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The :class:`~mne.io.Raw` object and the events array are the bare minimum
+# needed to create an :class:`~mne.Epochs` object, which we create with the
+# :class:`mne.Epochs` class constructor. Here we'll also specify some data
+# quality constraints: we'll reject any epoch where peak-to-peak signal
+# amplitude is beyond reasonable limits for that channel type. This is done
+# with a *rejection dictionary*:
+
+reject_criteria = dict(mag=4000e-15,     # 4000 fT
+                       grad=4000e-13,    # 4000 fT/cm
+                       eeg=150e-6,       # 150 μV
+                       eog=250e-6)       # 250 μV
+
+###############################################################################
+# We'll also pass the event dictionary as the ``event_id`` parameter (so we can
+# work with easy-to-pool event labels instead of the integer event IDs), and
+# specify ``tmin`` and ``tmax`` (the time relative to each event at which to
+# start and end each epoch). By default, :class:`~mne.io.Raw` and
+# :class:`~mne.Epochs` data aren't loaded into memory (they're accessed from
+# disk only when needed), but here we'll force loading into memory so that we
+# can see the results of the rejection criteria being applied:
+
+epochs = mne.Epochs(raw, events, event_id=event_dict, tmin=-0.2, tmax=0.5,
+                    reject=reject_criteria, preload=True)
+
+###############################################################################
+# .. note::
```

Comment:
Looking at the rendered version, I find these boxes a bit distracting. Perhaps it's because of too many colors. I think it should be in the same color as the code (or another shade of gray). But this might be a PR of its own

Reply:
I'll switch them to sidebars, hopefully that's less distracting. I agree that the colors can be a little distracting (esp. when the red preformatted text gets a light-gray background on top of the blue background for the rest of the box).

Reply:
actually, sidebars aren't working well here (they get covered up by the code/output blocks) so I've incorporated the content of one note into the main text, and moved the other note to the end of a section so it's less of an interruption.  @larsoner I tried adding a `z-index: 100;` to `div.sidebar` in `doc/_static/styles.css` but it didn't fix it; any other ideas?